lib/config-generator.js

```diff
@@ -149,13 +149,23 @@ class ConfigGenerator {
                 filename = this.webpackConfig.configuredFilenames.images;
             }
 
+            // The url-loader can be used instead of the default file-loader by
+            // calling Encore.configureUrlLoader({ images: {/* ... */}})
+            let loaderName = 'file-loader';
+            let loaderOptions = {
+                name: filename,
+                publicPath: this.webpackConfig.getRealPublicPath()
+            };
+
+            if (this.webpackConfig.urlLoaderOptions.images) {
+                loaderName = 'url-loader';
+                loaderOptions = this.webpackConfig.urlLoaderOptions.images;
```

Comment:
the file-loader options must be combined with the url-loader options. Otherwise, we won't have the right config in case the url-loader fallbacks to the file-loader for bigger files.

Reply:
Definitely, should be good now!